### PR TITLE
[plugin.video.roosterteeth] 1.3.1

### PR DESCRIPTION
**Pull request by skipmodea1**

### CODE REVIEW

File: plugin.video.roosterteeth/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Auto detect text files and perform LF normalization
```
Comment (learningit) [inline plugin.video.roosterteeth/.gitattributes]: Please remove the gitattributes and gitignore files from the commit.

Comment (skipmodea1) [inline plugin.video.roosterteeth/.gitattributes]: removed the gitattributes and gitignore files.

**Comment by skipmodea1** [comment]
I removed the gitattributes and gitignore files.

**Comment by learningit** [comment]
I missed this before: I think you should be using xbmc.LOGDEBUG rather than xbmc.LOGNOTICE for your debug messages. The rest looks OK to me.

**Comment by skipmodea1** [comment]
Changed debug stuff

**Pull request merged**